README.md

```diff
@@ -102,17 +97,28 @@ let client = try Client.from(bundle: keys, options: .init(api: .init(env: .produ
 
 You can configure the client's network connection and key storage method with these optional parameters of `Client.create`:
 
-| Parameter | Default | Description                                                                                                                                                                                                                                                                     |
+| Parameter | Default | Description |
 | --------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| env       | `dev`   | Connect to the specified XMTP network environment. Valid values include `.dev`, `.production`, or `.local`. For important details about working with these environments, see [XMTP `production` and `dev` network environments](#xmtp-production-and-dev-network-environments). |
+| env       | `dev`   | Connect to the specified XMTP network environment. Valid values include `.dev`, `.production`, or `.local`. For important details about working with these environments, see [XMTP `production` and `dev` network environments](#xmtp-production-and-dev-network-environments).        |
+
+#### Configure `env`
 
 ```swift
 // Configure the client to use the `production` network
 let clientOptions = ClientOptions(api: .init(env: .production))
 let client = try await Client.create(account: account, options: clientOptions)
 ```
 
-**Note: The `apiUrl`, `keyStoreType`, `codecs`, `maxContentSize` and `appVersion` parameters from the XMTP client SDK for JavaScript (xmtp-js) are not yet supported.**
+## Configure content types
+
+You can use custom content types by calling `Client.register`. The SDK comes with two commonly used content type codecs, `AttachmentCodec` and `RemoteAttachmentCodec`:
+
+```swift
+Client.register(AttachmentCodec())
+Client.register(RemoteAttachmentCodec())
+```
+
+To learn more about using `AttachmentCodec` and `RemoteAttachmentCodec`, see [Handle different content types](#handle-different-content-types).
```

Comment:
@nakajima I added this link to take folx to the deeper discussion on how to use these codecs. Does that make sense? <3